ui/userMenu.js

```diff
@@ -0,0 +1,306 @@
+            this._sessionModeUpdatedId = 0;
+        }
+    }
+
+    _createPanelIcon() {
+        this.panelBox = new St.BoxLayout({
+            x_align: Clutter.ActorAlign.CENTER,
+            y_align: Clutter.ActorAlign.CENTER,
+        });
+        this._panelAvatar = new UserWidget.Avatar(this._user, {
+            iconSize: USER_ICON_SIZE,
+            styleClass: 'user-menu-button-icon',
+            reactive: true,
+        });
+        this.panelBox.add_actor(this._panelAvatar);
+    }
+
+    _createPopupMenu() {
+        this.menu = new PopupMenu.PopupMenuSection();
+
+        this._accountSection = new UserAccountSection(this._user);
+        this._activateId = this._accountSection.userIconItem.connect('activate', () => {
+            this._launchApplication(USER_ACCOUNTS_PANEL_LAUNCHER);
+        });
+
+        this.menu.addMenuItem(this._accountSection);
+        this.menu.addMenuItem(new PopupMenu.PopupSeparatorMenuItem());
+
+        let menuItemsSection = new PopupMenu.PopupMenuSection();
+        menuItemsSection.box.style_class = 'user-menu-items';
+
+        // Control Center
+        let gicon = new Gio.ThemedIcon({ name: 'applications-system-symbolic' });
+        this._settingsItem = menuItemsSection.addAction(SETTINGS_TEXT, () => {
+            this._launchApplication(SETTINGS_LAUNCHER);
+        }, gicon);
+
+        // Social
+        gicon = new Gio.ThemedIcon({ name: 'user-available-symbolic' });
+        let onlineAccountsItem = menuItemsSection.addSettingsAction(ONLINE_ACCOUNTS_TEXT,
+            ONLINE_ACCOUNTS_PANEL_LAUNCHER);
+        // PopupMenuSection.addSettingsAction doesn't support an icon, let's
+        // add it manually
+        onlineAccountsItem.remove_child(onlineAccountsItem.label);
+        onlineAccountsItem.icon = new St.Icon({ style_class: 'popup-menu-icon',
+                                                x_align: Clutter.ActorAlign.END });
+        onlineAccountsItem.icon.gicon = gicon;
+        onlineAccountsItem.add_child(onlineAccountsItem.icon);
+        onlineAccountsItem.add_child(onlineAccountsItem.label);
+
+        let app = null;
+        let iconFile = null;
+
+        // Feedback
+        app = Shell.AppSystem.get_default().lookup_app(FEEDBACK_LAUNCHER);
+        if (app) {
+            iconFile = Gio.File.new_for_uri(
+                `file://${PanelExtension.path}/data/icons/feedback-symbolic.svg`);
+            gicon = new Gio.FileIcon({ file: iconFile });
+            menuItemsSection.addAction(FEEDBACK_TEXT, () => {
+                this._launchApplication(FEEDBACK_LAUNCHER);
+            }, gicon);
+        }
+
+        // Help center
+        app = Shell.AppSystem.get_default().lookup_app(HELP_CENTER_LAUNCHER) ||
+              Shell.AppSystem.get_default().lookup_app(HELP_CENTER_LAUNCHER_ALT);
+        if (app) {
+            iconFile = Gio.File.new_for_uri(
+                `file://${PanelExtension.path}/data/icons/endless-help-symbolic.svg`);
```

Comment:
I'm not sure if gettext is able to properly handle translations when backticks are present in the source file, this needs double checking

Reply:
I guess this comment came to the wrong line? There is no translatable string involved here but good general point anyway, will check for occurrences.

Reply:
Checked and couldn't find any place where we use backticks for translatable strings, please let me know if I am missing something.